### ISSUE TITLE
add arbitrary tags to grid squares

### ISSUE THREAD

**Issue by mikelmaron** [hotosm/osm-tasking-manager#136]
could possibly be a solution to both https://github.com/hotosm/osm-tasking-manager/issues/134 and https://github.com/hotosm/osm-tasking-manager/issues/130

**Comment by dvenza** [comment]
I'm looking into this, keeping in mind #134, #130 and #162.

**Comment by dvenza** [comment]
The problem with tags is that showing them on the map inside the tile is going to be confusing.
On the other hand, showing them in the task panel means that the user will see them only after the task was taken, and that's a bit pointless.

**Comment by pgiraud** [comment]
Hey @dvenza, thanks for your help.
Can you please first have a try at the workflow branch which is in my repo?
It should be merged into master sooner than later and it would be a good option to base your developments on this branch.
Also I got interesting feedback regarding the new interface to which we should pay attention too.

Before coding anything, if you have mockups to share please don't hesitate. I would love to discuss the interface.

**Comment by pgiraud** [comment]
Regarding your last comment, I advice you to take a look at the new interface.
Users can access to tiles in a read-only mode. http://tasks-dev.hotosm.org/job/361#task/6899/4349/13

**Comment by pgiraud** [comment]
There's one thing that is missing in the new interface: the countdown ("You have xx minutes left" ...).

**Comment by dvenza** [comment]
Ok! Thanks, I'll look at it.
